scripts/gaussian_codegen/pdf/derive.py

```
@@ -166,15 +166,15 @@ def main(argv: Sequence[str] | None = None) -> int:
 
     # Coefficient magnitude check at the PDF accumulation scale before serialization.
-    wad = mpf(constants.PDF_WAD)
+    acc_scale = mpf(constants.PDF_ACC_SCALE)
     max_mag = mpf(0)
     for c in num + den:
-        m = abs(c) * wad
+        m = abs(c) * acc_scale
         if m > max_mag:
             max_mag = m
-    print(f"Max |coeff| × PDF_WAD = {float(max_mag):.3e} (must be < 2^128 ≈ 3.4e38)")
+    print(f"Max |coeff| × PDF_ACC_SCALE = {float(max_mag):.3e} (must be < 2^128 ≈ 3.4e38)")
```

Ruff (0.15.21)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)